File: RestClient/RestCiguaClient/RestCiguaClient/Entidades/tweetRest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RestCiguaClient.Entidades
{
    class tweetRest
    {
        public string contenido { get; set; }
        public string imagenNombre { get; set; }
        public byte[] imagen { get; set; }
        public bool isPublic { get; set; }
        public string username { get; set; }

        public tweetRest()
        {

        }
    }
}

File: src/java/edu/pucmm/pw/parcial2/entidades/tweetRest.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package edu.pucmm.pw.parcial2.entidades;

import java.io.Serializable;

/**
 *
 * @author DavidA
 */
public class tweetRest implements Serializable{
    
    public String contenido;
    public String imagenNombre;
    public byte[] imagen;
    public boolean isPublic;
    public String username;
    
    public tweetRest(){
    
    }
    
}
